### PR TITLE
test(PlaywrightNewScreenshot): added the expectScreenshot fixture

### PR DESCRIPTION
**Pull request by NasgulNexus**
added the expectScreenshot fixture, which makes it possible to take screenshots in light and dark themes

### CODE REVIEW

**Comment by amje** [comment]
There are double dashes in snapshot names

**Comment by NasgulNexus** [comment]
> @NasgulNexus There are double dashes in snapshot names

fixed the double dash

File: playwright/README.md [hunk 1]
```diff
@@ -6,23 +6,61 @@
 2. Inside the component folder, create the `__tests__` folder and create a file inside it with the following name `<ComponentName>.visual.test.tsx`
 3. Writing a test:
 
-   ```ts
-   import React from 'react';
+Take a screenshot by default, takes a screenshot in a light theme
```
Comment (amje) [inline playwright/README.md:9]:
```suggestion
Capture a screenshot, by default in light theme only:
```

File: playwright/README.md [hunk 2]
```diff
@@ -6,23 +6,61 @@
...
 
-   import {expect} from '@playwright/experimental-ct-react';
+```ts
+import React from 'react';
 
-   import {MyTestedComponent} from '../MyTestedComponent';
+import {expect} from '@playwright/experimental-ct-react';
 
-   import {test} from '~playwright/core';
+import {MyTestedComponent} from '../MyTestedComponent';
```
Comment (amje) [inline playwright/README.md:16]: Just `Component` would be ok, `Tested` means it's tested?

File: playwright/README.md [hunk 3]
```diff
@@ -6,23 +6,61 @@
...
-   test('Name test', async ({mount}) => {
-     //mounting a component
-     const component = await mount(<MyTestedComponent props={props} />);
+import {test} from '~playwright/core';
 
-     //screenshot
-     await expect(component).toHaveScreenshot();
-   });
-   ```
+test('Name test', async ({mount}) => {
```
Comment (amje) [inline playwright/README.md:20]:
```suggestion
test('test description', async ({mount}) => {
```

File: playwright/README.md [hunk 4]
```diff
@@ -6,23 +6,61 @@
...
+  //mounting a component
```
Comment (amje) [inline playwright/README.md:21]:
```suggestion
  // mount the component
```

File: playwright/README.md [hunk 5]
```diff
@@ -6,23 +6,61 @@
...
+  const component = await mount(<MyTestedComponent props={props} />);
+
+  //screenshot
```
Comment (amje) [inline playwright/README.md:24]:
```suggestion
  // capture the screenshot
```

File: playwright/README.md [hunk 6]
```diff
@@ -6,23 +6,61 @@
...
+  await expect(component).toHaveScreenshot();
+});
+```
+
+You can also take screenshots in dark and light themes
```
Comment (amje) [inline playwright/README.md:29]:
```suggestion
You can also capture screenshots both in dark and light themes:
```

File: playwright/README.md [hunk 7]
```diff
@@ -6,23 +6,61 @@
...
+test('Name test', async ({mount, expectScreenshot}) => {
+  //mounting a component
+  await mount(<MyTestedComponent props={props} />);
+
+  //screenshot
+  await expectScreenshot();
+});
+```
+
+or if you need to do any actions on the component
```
Comment (amje) [inline playwright/README.md:47]:
```suggestion
If you need to do any actions with the component:
```

File: playwright/core/expectScreenshotFixture.ts [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+            });
+        };
+
+        const nameScreenshot = testInfo.titlePath.slice(1).join(' ');
+
+        expect(await captureScreenshot('g-root_theme_dark')).toMatchSnapshot({
+            name: `${screenshotName || nameScreenshot} dark.png`,
+        });
+
+        expect(await captureScreenshot('g-root_theme_light')).toMatchSnapshot({
```
Comment (amje) [inline playwright/core/expectScreenshotFixture.ts:34]: Better to pass `light` or `dark`, and use it inside `captureScreenshot` like so: `g-root_theme_${theme}`

**Review by amje — CHANGES_REQUESTED** [review]

**Review by amje — APPROVED** [review]

**Review by korvin89 — APPROVED** [review]

**Pull request merged**